docs/sdk/README.md

```diff
@@ -13,7 +13,7 @@ The Tableland SDK provides a minimal client that implements the Cloudflare D1 D
 
 ## Synopsis
 
-The `@tableland/sdk` comes with a few APIs: "primary" [Cloudflare D1 client compatible](https://developers.cloudflare.com/d1/platform/client-api/) API for [`Database`](/sdk/database) connections, and two additional "secondary" APIs ([`Validator`](/sdk/validator) and [`Registry`](/sdk/registry)) that are unique to Tableland operations. Generally, users will leverage the primary database API for typical SQL operations, and the secondary APIs are useful when needing to interact directly with a Tableland Validator node or directly calling Tableland’s smart contracts (on-chain access control, direct contract calls, etc.).
+The `@tableland/sdk` comes with a few APIs: "primary" [Cloudflare D1 client compatible](https://developers.cloudflare.com/d1/platform/client-api/) API for [`Database`](/sdk/database) connections, and two additional "secondary" APIs ([`Validator`](/sdk/validator) and [`Registry`](/sdk/registry)) that are unique to Tableland operations. Generally, users will leverage the primary database API for typical SQL operations, and the secondary APIs are useful when needing to interact directly with a Tableland Validator node or directly calling Tableland’s smart contracts (on-chain access control, direct contract calls, etc.). Both ESM and CommonJS are supports.
```

Comment:
I am I reading this wrong, or should it be "Both ESM and CommonJS are supported."?

Reply:
ah good catch, typo